Documentation/EyeTracking/EyeTracking_TargetSelection.md

```diff
@@ -1,26 +1,65 @@
 ![MRTK](../Images/EyeTracking/mrtk_et_targetselect.png)
 
-# Eye-Supported Target Selection
+# Eye-supported target selection
 This page discusses different options for accessing eye gaze data and eye gaze specific events to select targets in MRTK. 
-Eye Tracking allows for fast and effortless target selections using a combination of information about what a user is looking at with additional inputs such as 
-_hand tracking_ and _voice commands_:
-    - Look & Pinch (i.e., hold up your hand in front of you and pinch your thumb and index finger together)
-    - Look & Say _"Select"_ (default voice command)
-    - Look & Say _"Explode"_ or _"Pop"_ (custom voice commands)
-    - Look & Bluetooth button
-
-## Target Selection
-### Use Generic Focus Handler 
-If Eye Tracking is set up correctly (see [Basic MRTK Setup to use Eye Tracking](EyeTracking_BasicSetup.md)), enabling users to select 
+Eye tracking allows for fast and effortless target selections using a combination of information about what a user is looking at with additional inputs such as _hand tracking_ and _voice commands_:
+- Look & Say _"Select"_ (default voice command)
+- Look & Say _"Explode"_ or _"Pop"_ (custom voice commands)
+- Look & Bluetooth button
+- Look & Pinch (i.e., hold up your hand in front of you and bring your thumb and index finger together)
+    - _Please note that for this to work, the [hand rays need to be disabled](EyeTracking_EyesAndHands.md)!_
```

Comment:
Following up on another comment I had above, please have this directly link to the section describing how to do this - otherwise people might not find the right section/information.

Reply:
Again - the page is pretty short, but yes, I updated it to the following now: EyeTracking_EyesAndHands.md#how-to-disable-the-hand-ray.